### PR TITLE
Add UI for adding / editing Global Rules

### PR DESCRIPTION
**Pull request by mconnell**
* Adds the ability to add and edit global rules
* name refactor of RulesController to LocalRulesController
* Added a helper to create a description for rules

### CODE REVIEW

**Comment by benbarnett** [comment]
:+1:

**Comment by munckymagik** [comment]
:+1:

File: app/assets/stylesheets/_table.scss [hunk 1]
```diff
@@ -59,3 +59,9 @@ th > p {
   font-weight: 500;
   // NOTE that this is here to override sloppy formatting in the tables where p tags is not bolded.
 }
+
+td ul {
```
Comment (benbarnett) [inline app/assets/stylesheets/_table.scss]: :( at this style

**Comment by benbarnett** [comment]
Please ask for another review if you're going to add commits after getting a thumb.

**Comment by urfolomeus** [comment]
Sorry, but this PR was holding us up from progressing any further. You are right though I should not have closed before checking with you. Can you help me to do what I want to do with this style wise please?

**Comment by benbarnett** [comment]
That is not a reason just to merge any old PR, sorry. You should use a proper BEM naming convention or a layout class.

**Comment by urfolomeus** [comment]
Point taken. I'm sorry.

**Comment by urfolomeus** [comment]
@benbarnett should I add the new BEM rule into the _table.css file or create a new file specifically for this table?

**Comment by benbarnett** [comment]
@urfolomeus have a read through our [frontend coding standards](https://github.com/moneyadviceservice/frontend-code-standards#bem-naming-convention) and see what makes sense in this situation. The blog post it links to on CSS Wizardry is also well worth a read.

**Pull request merged**